### ISSUE TITLE
Energy loss function discrepancies

### ISSUE THREAD

**Issue by astronomike** [Hyperthetical/DarkMatters#11]
The energy loss functions in `electron.c` and `cn_electron.py` are different - the green's method assumes the input array comes in as gammas, whereas I had the crank version coming in as energies. The crank version also doesn't choose between the ISRF being switched on/off. 

This is the difference between the two methods as it stands (lossOnly set to True):
![SB_lossOnlycoma_nu10](https://user-images.githubusercontent.com/5190106/167018100-38b9e81d-713a-4a55-88f4-c2aa045cc8ff.png)

After a quick change of the above things (converting crank energy loss function to gamma input array, and correctly accounting for ISRF) in `cn_electron.py`:
```python
    def set_b(self,B,ne,E,ISRF=False):
        #set and return energy loss function [GeV s^-1]
        b = self.constants
        me = 0.511e-3       #[me] = GeV/c^2 
        if ISRF:
            eloss = b['ICISRF']*(E*me)**2 + b['sync']*(E*me)**2*B**2 + b['coul']*ne*(1+np.log(E/ne)/75.0) + b['brem']*ne*E*me
        else:
            eloss = b['ICCMB']*(E*me)**2 + b['sync']*(E*me)**2*B**2 + b['coul']*ne*(1+np.log(E/ne)/75.0) + b['brem']*ne*E*me

        self.b = eloss/me
       
        return eloss/me
```
This is the output: 
![SB_lossOnly_newbfunccoma_nu10](https://user-images.githubusercontent.com/5190106/167018431-e304f6e6-4ff5-4488-8a66-b99445d8a98a.png)

This looks a bit better, but I'm unhappy with a few things. Firstly, just printing out the energy loss vectors calculated with each method shows there's still a fair amount of discrepancy - it's a little difficult to pinpoint this since they have different shapes, and are based on average quantities for greens but not for crank. Do you think this is expected? 

Secondly, and the part I am most confused about, is the values for the SB at different frequencies in the crank method just don't behave at all like the greens ones. Here is an animation of this:
 
![SB_lossOnly_newbfunccoma](https://user-images.githubusercontent.com/5190106/167019694-cc9ba177-5e9b-459f-a2a3-f4015b572ff8.gif)

The spread of values over the WIMP masses is also a little different. It seems like there is some unexpected dependence on WIMP mass with this implementation that I'm not sure about. 

Could you please take a look at this and see if you can help me spot the issue? 

Another option that I was thinking of would be to just import the energy loss functions defined in `electron.py` for use with the crank method as well. Do you think this is preferable? If we can deal with the reshaping of the arguments, we could bypass the differing implementations of these functions.
